resources-ktx/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - **Text**: added abstract methods `equals`, `hashCode`
 - **Text**: added `Parcelable` implementation
+- **Text**: added `Companion object` and `Text.EMPTY` implementation
```

Comment:
I suggest making the first letter lowercase
```suggestion
- **Text**: added `companion object` and `Text.EMPTY` implementation
```

Reply:
Done 👍